requirements.txt

```diff
@@ -8,4 +8,4 @@ gpsoauth==0.3.0
 coveralls==1.1
 werkzeug==0.11.10
 sqlalchemy==1.0.14
--e git+https://github.com/tejado/pgoapi.git@0811db23d639039f968a82e06c7aa15a0a5016b6#egg=pgoapi
+-e git+https://github.com/keyphact/pgoapi.git#egg=pgoapi
```

Comment:
I get this error when i try to install the new API:
https://i.gyazo.com/4d2a80e943d1ee6f12823dfb2ca5f448.png